suite-native/module-accounts-import/src/accountsImportThunks.ts

```diff
@@ -116,6 +119,21 @@ export const getAccountInfoThunk = createThunk<
             ]);
 
             if (fetchedAccountInfo?.success) {
+                const tokenDefinitions = selectNetworkTokenDefinitions(getState(), networkSymbol);
+
+                // fetch token definitions for this network in case they are needed
+                if (!tokenDefinitions) {
+                    const definitionTypes = getSupportedDefinitionTypes(networkSymbol);
+
+                    definitionTypes.forEach(async type => {
+                        await dispatch(
```

Comment:
This happens when there is a loading screen present and user is already waiting. It feels to be the best moment in the flow. 

Also only token definitions for imported network are being fetched, no other networks

Reply:
Still think it will be better to start sooner, it could run in background and have more time to finish. Also you are not awaiting it here so there is no guarantee that it will be fetched b when next screen is displayed

Reply:
I don't have strong onion, so you may keep it here but at fix that await

Reply:
It is already awaited in f7d2969